airflow/www/static/js/dag/details/taskInstance/ExtraLinks.tsx

```diff
@@ -57,8 +57,15 @@ const ExtraLinks = ({
     if (!url) {
       return true;
     }
-    const urlRegex = /^(https?:)/i;
-    return urlRegex.test(url);
+    const path = new URL(url, "http://localhost");
```

Comment:
One more question. Will relative paths work with just localhost? Do we not need to include the port number?

Reply:
<img width="292" alt="Screenshot 2024-09-04 at 17 51 21" src="https://github.com/user-attachments/assets/0745749f-5391-45f5-90d3-cc16e30e5f6a">

Reply:
Exactly, it would have to be something like `http://localhost:8080/test`to be valid

Reply:
Well. It's quite valid actually - you can omit port, no problem with it.

Reply:
(and we are just doing this to parse the UIRL and see if it's "correct")

Reply:
I think it should be fine without 8080 since we are just validating the URL, right?

Reply:
Yeah. I think so @bbovenzi ?

Reply:
@utkarsharma2
I believe we should use the value from AIRFLOW__WEBSERVER__BASE_URL instead of localhost, and this change needs to be backported to version 2.10